_install-and-configure/upgrade-opensearch/index.md

```diff
@@ -137,6 +137,31 @@ td {
         <th>OpenSearch Version</th>
         <th>Elasticsearch Version</th>
     </tr>
+    <tr>
+        <td>9.10.0</td>
+        <td>2.13.0</td>
+        <td>8.13</td>
+    </tr>
+    <tr>
+        <td>9.8.0</td>
+        <td>2.12.0</td>
```

Comment:
Looks like 2.12 shipped with 9.9.2: https://github.com/opensearch-project/OpenSearch/blob/2.12.0/buildSrc/version.properties

No released version of OpenSearch used 9.8

Reply:
Our release notes for 2.12 claim we did. Sorry I went through the release notes because it was quicker than combing through all the manifests.

Reply:
https://github.com/opensearch-project/OpenSearch/releases/tag/2.12.0